README.md

```diff
@@ -2,6 +2,24 @@
 * Spring filter that adds zone and correlation id to log
```

Comment:
This filter initializes a HTTP header(X-B3-TraceID) for tracing, if not already present. The header is also added in the outgoing response.